platform/security/claims/security-claims-attributequery/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -46,8 +46,8 @@
         <property name="simpleSign" ref="simpleSign"/>
         <property name="issuer" value=""/>
         <property name="destination" value=""/>
-        <property name="signatureProperties" ref="signatureProperties"/>
-        <property name="encryptionProperties" ref="encryptionProperties"/>
+        <property name="signaturePropertiesLocation" value="${ddf.etc}/ws-security/server/signature.properties"/>
+        <property name="encryptionPropertiesLocation" value="${ddf.etc}/ws-security/server/encryption.properties"/>
```

Comment:
✏️ Nit: Can these be arguments instead of properties since they'll never change, right?

Reply:
The entire class uses properties, so I'm going to leave it as is for now.

Reply:
My suggestion was just a nit, but by arguments, I meant constructor arguments, as opposed to setting the fields via setters. Constructor arguments would allow the fields to be `final`, and the fields could still be used by the entire class.